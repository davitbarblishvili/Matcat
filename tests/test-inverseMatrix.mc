func main() int 
{
    matrix a; 
    matrix c; 
    a = [[1,2],[4,5]];
    inv(a);
    
    return 0;

}