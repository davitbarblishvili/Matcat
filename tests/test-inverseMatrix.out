(
[-1.67 0.67 ]
[1.33 -0.33 ]
)